package.json

```diff
@@ -25,7 +25,7 @@
     "ws": "^7.4.2"
   },
   "peerDependencies": {
-    "zwave-js": "^6.2"
+    "zwave-js": "^6.3"
```

Comment:
Let's do separate PRs for bumping zwave-js, the log feature addition and bumping the server version.